sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -186,7 +185,7 @@ public ConfigurationClientBuilder() {
      * and {@link #retryPolicy(HttpPipelinePolicy)} have been set.
      */
     public ConfigurationClient buildClient() {
-        return new ConfigurationClient(buildInnerClient(), DEFAULT_SYNC_TOKEN_POLICY);
+        return new ConfigurationClient(buildInnerClient(), new SyncTokenPolicy());
```

Comment:
Does the `SyncTokenPolicy` used in the pipeline and the one passed to the ConigurationClient need to be the same?

Reply:
SyncTokenPolicy needs to be the same object.  Updated.